platforms/ios/demo/src/MapViewController.m

```diff
@@ -18,6 +18,15 @@ - (void)addAlert:(NSString *)message withTitle:(NSString *)title;
 
 @implementation MapViewControllerDelegate
 
+- (void)mapView:(TGMapViewController *)mapView didCompleteSceneUpdatesWithErrors:(NSArray<TGSceneUpdateStatus *> *)updateStatuses
+{
+    for (TGSceneUpdateStatus* updateStatus in updateStatuses) {
+        NSLog(@"Scene update error for update with path %@ and value %@",
```

Comment:
Sounds good, I understand more the use cases of `NSError` and how it'll be used, I'll look how this can fit nicely with this interface.